### ISSUE TITLE
Null check operator used on a null value

### ISSUE THREAD

**Issue by lhimo** [tomaszpolanski/faster_driver#17]
Launching `fasterdriver ./integration_test` command gives this error:
```
Unhandled exception:
Null check operator used on a null value
#0      TestWriter.generateMainTest (package:faster_driver/src/test_writer.dart:22:59)
#1      run (package:faster_driver/faster_driver.dart:20:46)
#2      main (file:///Users/foo/.pub-cache/hosted/pub.dartlang.org/faster_driver-0.1.2/bin/main.dart:12:9)
#3      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:281:32)
#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
```

I'm using Flutter 2.5.3 on Apple Silicon (M1) processor if it helps.

**Comment by tomaszpolanski** [comment]
Thanks, @lhimo for the report, I will try to reproduce it

**Comment by tomaszpolanski** [comment]
I've missed in one place error reporting but the path you are passing does not exit - I will properly display error message but meanwhile check out the [example](https://github.com/tomaszpolanski/faster_driver/tree/master/example)

**Comment by lhimo** [comment]
Thanks for checking! The path actually exists and contains a few `*_test.dart` files. Not sure if it's something to do with the M1 processor or with the installation, but I followed the example

**Comment by tomaszpolanski** [comment]
I will soon update the package to show proper info if the given path is not found - then there should be more info to debug to see why in your case the folder and/or files seemed to be missing

**Comment by lhimo** [comment]
The error actually refers to the `fileName` (which defaults to main_tests.dart), not to the `directory` parameter which contain all the tests

**Comment by tomaszpolanski** [comment]
Is that file missing for real or it is available but reported wrongly?
During the weekend I will have time to fix reporting issue or if the file is actually there, fix it

**Comment by lhimo** [comment]
Got it, the issue is gone if I create a `main_tests.dart` empty file manually before executing the script

**Comment by tomaszpolanski** [comment]
The issue should be now fixed in `0.1.2+1`, thanks for reporting :)
